.github/workflows/test-e2e.yml

```diff
@@ -155,13 +155,13 @@ jobs:
           done
       - name: Checkout template-operator
         uses: actions/checkout@v3
-        if: ${{ matrix.e2e-test == 'module-status-propagation' ||  matrix.e2e-test == 'module-without-default-cr' ||  matrix.e2e-test == 'non-blocking-deletion' ||  matrix.e2e-test == 'purge-controller' ||  matrix.e2e-test == 'purge-metrics' }}
+        if: ${{ matrix.e2e-test == 'module-status-propagation' ||  matrix.e2e-test == 'module-without-default-cr' ||  matrix.e2e-test == 'non-blocking-deletion' ||  matrix.e2e-test == 'purge-controller' ||  matrix.e2e-test == 'purge-metrics' || matrix.e2e-test == 'kyma-deletion'}}
```

Comment:
which one doesn't need to check out template-operator? We can change to if not equals, this if condition is too long. Or even remove this if, actually doesn't matter have this checkout step if not needed.